AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -110,13 +110,16 @@ def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size
         axon_diam = prop_axon.equivalent_diameter * pixelsize
         # Axon area in µm^2
         axon_area = prop_axon.area * (pixelsize ** 2)
+        # Axon perimeter in micrometers
+        axon_perimeter = prop_axon.perimeter * pixelsize
```

Comment:
In terms of the myelin, is this the only the inner perimeter? If so, it should be documented here for clarity I think.

Reply:
Fixed